src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  padding-left: var(--pf-c-scrollspy--PaddingLeft);
+  background-color: var(--pf-c-scrollspy--BackgroundColor);
+
+  &.pf-m-background {
+    --pf-c-scrollspy--BackgroundColor: var(--pf-c-scrollspy--m-background--BackgroundColor);
+  }
+
+  &.pf-m-center {
+    text-align: center;
+  }
+
+  &.pf-m-vertical {
+    --pf-c-scrollspy__list--PaddingRight: 0;
+    --pf-c-scrollspy__list--PaddingLeft: 0;
+    --pf-c-scrollspy__list--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderTopWidth);
+    --pf-c-scrollspy__list--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth);
+    --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--focus--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth);
+    --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: var(--pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth);
+
+    .pf-c-scrollspy__list {
+      flex-direction: column;
+    }
+  }
+}
+
+.pf-c-scrollspy__list {
+  position: relative;
+  display: inline-flex;
+  padding-right: var(--pf-c-scrollspy__list--PaddingRight);
+  padding-left: var(--pf-c-scrollspy__list--PaddingLeft);
+  margin: 0;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border-color: var(--pf-c-scrollspy__list--before--BorderColor);
+    border-style: solid;
+    border-width: var(--pf-c-scrollspy__list--before--BorderTopWidth) var(--pf-c-scrollspy__list--before--BorderRightWidth) var(--pf-c-scrollspy__list--before--BorderBottomWidth) var(--pf-c-scrollspy__list--before--BorderLeftWidth);
+  }
+}
+
+.pf-c-scrollspy__link {
+  position: relative;
+  padding-top: var(--pf-c-scrollspy__link--PaddingTop);
+  padding-right: var(--pf-c-scrollspy__link--PaddingRight);
+  padding-bottom: var(--pf-c-scrollspy__link--PaddingBottom);
+  padding-left: var(--pf-c-scrollspy__link--PaddingLeft);
+  border: 0;
+  outline-offset: var(--pf-c-scrollspy__link--OutlineOffset);
+
+  &:hover {
+    --pf-c-scrollspy__item-text--Color: var(--pf-c-scrollspy__link--hover__item-text--Color);
+  }
+
+  &:focus {
+    --pf-c-scrollspy__item-text--Color: var(--pf-c-scrollspy__link--focus__item-text--Color);
+    --pf-c-scrollspy__link--before--BorderTopWidth: var(--pf-c-scrollspy__link--focus--before--BorderTopWidth);
+    --pf-c-scrollspy__link--before--BorderLeftWidth: var(--pf-c-scrollspy__link--focus--before--BorderLeftWidth);
+    --pf-c-scrollspy__link--before--BorderColor: var(--pf-c-scrollspy__link--focus--before--BorderColor);
+  }
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
```

Comment:
You don't need this declaration

Reply:
when i removed this the current border wasnt being applied